pages/en/supported-networks/arweave.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: Building Subgraphs on Arweave 
+---
+
+> Arweave support in Graph Node and on the Hosted Service is in beta: please reach us on [Discord](https://discord.gg/rC8rBuRtbH) with any questions about building Arweave subgraphs!
+
+In this guide, you will learn how to build and deploy Subgraphs to index the Arweave blockweave.
```

Comment:
I'm curious about usage of `blockweave`, we should probably `blockchain` here and stay consistent and not use the "special" Arweave name, IMO.

```suggestion
In this guide, you will learn how to build and deploy Subgraphs to index the Arweave blockchain.
```